### PR TITLE
Newsletter 08/2020: Review

### PR DESCRIPTION
**Pull request by wurstsalat3000**
@Echolon

### CODE REVIEW

File: content/posts/newsletter/2020-08-08.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Translations of the XMPP Newsletter will be released here:
 
 ## Articles
 
-Lorenz Kästle wrote an article on the setup for a [Icinga 2 system with an python3-slixmpp library](https://www.netways.de/blog/2020/08/28/zurueck-in-die-zukunft-icinga2-benachrichtigungen-mit-xmpp/). See the [reposity on Github](https://github.com/RincewindsHat/icinga2_xmpp-notifications), too.
+Lorenz Kästle wrote an article on [how to setup an Icinga 2 system using python3-slixmpp](https://www.netways.de/blog/2020/08/28/zurueck-in-die-zukunft-icinga2-benachrichtigungen-mit-xmpp/) ([reposity on Github](https://github.com/RincewindsHat/icinga2_xmpp-notifications)).
```
Comment (pmaziere) [inline content/posts/newsletter/2020-08-08.md:22]: an article => an article in german

Comment (Echolon) [reply content/posts/newsletter/2020-08-08.md:22]: German

**Comment by Echolon** [comment]
@wurstsalat3000 can you review it?

**Comment by Echolon** [comment]
Thanks! Maybe the numerical Newsletter version is really better...

**Pull request merged**